lib/Utils/Backends.pm

```diff
@@ -47,7 +47,7 @@ our %EXPORT_TAGS = (
 sub use_ssh_serial_console {
     select_console('root-ssh');
     $serialdev = 'sshserial';
-    set_var('SERIALDEV', $serialdev);
```

Comment:
I actually mean here, SERIALDEV should be floating as it needs to be, eg here when use ssh serial console, the SERIALDEV should be set to sshserial.